src/Controllers/WorkerController.php

```diff
@@ -77,9 +77,11 @@ public function queue(Request $request, WorkerInterface $worker, Container $lara
             ]
         ]);
 
+        $bodyArray = json_decode($body, true);
+
         $worker->process(
             $request->header('X-Aws-Sqsd-Queue'), $job, [
-                'maxTries' => 0,
+                'maxTries' => (isset($bodyArray['maxTries']) && !is_null($bodyArray['maxTries']))? $bodyArray['maxTries'] : 0,
```

Comment:
Can use a simple null coalescing operator here:

```suggestion
                'maxTries' => $bodyArray['maxTries'] ?? 0,
```